rust/crd/src/constants.rs

```diff
@@ -87,3 +87,7 @@ pub const LISTENER_VOLUME_NAME: &str = "listener";
 pub const LISTENER_VOLUME_DIR: &str = "/stackable/listener";
 
 pub const HDFS_UID: i64 = 1000;
+
+pub const DEFAULT_NAME_NODE_SECRET_LIFETIME: Duration = Duration::from_days_unchecked(7);
+pub const DEFAULT_DATA_NODE_SECRET_LIFETIME: Duration = Duration::from_days_unchecked(7);
+pub const DEFAULT_JOURNAL_NODE_SECRET_LIFETIME: Duration = Duration::from_days_unchecked(7);
```

Comment:
Can we please move them into `impl NameNodeConfigFragment` to prevent accidental use?

Reply:
[done](https://github.com/stackabletech/hdfs-operator/pull/619/commits/9245f3d68f284c398f752ca11f7860585bdc0cf3)